webapp/stores/team_store.jsx

```diff
@@ -321,6 +321,14 @@ class TeamStoreClass extends EventEmitter {
         }
     }
 
+    subtractUnread(teamId, msgs, mentions) {
```

Comment:
Should we really be subtracting? I thought MsgCount on the server only went up? Wouldn't it be better to just reset the count to match what the team has? Maybe I'm missing something but I feel like this could get out of sync too easily